app/SystemTrayService.ts

```diff
@@ -181,7 +181,9 @@ export class SystemTrayService {
       if (!browserWindow) {
         return;
       }
-      if (!browserWindow.isVisible()) {
+      if (browserWindow.isVisible()) {
+        browserWindow.hide();
+      } else {
         browserWindow.show();
       }
       forceOnTop(browserWindow);
```

Comment:
Should this line be here? Or should it be moved next to the `.show` call?

Reply:
Sure, it didn't seem to make a difference so I had left it there.